portability-api/src/main/java/org/dataportabilityproject/webapp/PortabilityApiMain.java

```diff
@@ -15,11 +15,13 @@
  */
 package org.dataportabilityproject.webapp;
 
+import com.google.common.util.concurrent.UncaughtExceptionHandlers;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 
 public class PortabilityApiMain {
   public static void main(String args[]) throws Exception {
+    Thread.setDefaultUncaughtExceptionHandler(UncaughtExceptionHandlers.systemExit());
```

Comment:
do we want to do this in prod? won't it disrupt other jobs being handled by this API instance, especially if they are in the API polling stage?

Reply:
Good point. I was debugging but in prod, we certainly don't want to exit. Added a warning message instead.